packages/grafana-runtime/src/unstable.ts

```diff
@@ -11,3 +11,10 @@
 
 // This is a dummy export so typescript doesn't error importing an "empty module"
 export const unstable = {};
+
+export {
+  ScopesContext,
+  type ScopesContextValueState,
+  type ScopesContextValue,
+  useScopes,
+} from './services/ScopesContext';
```

Comment:
Can we move this file out of services for now into an unstable directory or something so it doesn't accidentally find its way back into the index entrypoint please?